build/common.project.props

```
@@ -20,8 +20,8 @@
     <TargetFrameworksExeForSigning>$(TargetFrameworksExe);netcoreapp5.0</TargetFrameworksExeForSigning>
     <TargetFrameworksExeForSigning Condition=" '$(IsXPlat)' == 'true' ">$(NETCoreTargetFramework);netcoreapp5.0</TargetFrameworksExeForSigning>
     <TargetFrameworksExeForSigning Condition="'$(DotNetBuildFromSource)' == 'true'">$(TargetFrameworksExe)</TargetFrameworksExeForSigning>
-    <MinimalTargetFrameworksExeSigning>$(NETFXTargetFramework);netcoreapp5.0</MinimalTargetFrameworksExeSigning>
-    <MinimalTargetFrameworksExeSigning Condition=" '$(IsXPlat)' == 'true' ">netcoreapp5.0</MinimalTargetFrameworksExeSigning>
+    <MinimalTargetFrameworksExeSigning>$(NETFXTargetFramework);net7.0</MinimalTargetFrameworksExeSigning>
+    <MinimalTargetFrameworksExeSigning Condition=" '$(IsXPlat)' == 'true' ">net7.0</MinimalTargetFrameworksExeSigning>
```

Considering this version of NuGet inserts into the .NET 8 SDK, should we just bump straight to .NET 8 instead?

```suggestion
    <MinimalTargetFrameworksExeSigning>$(NETFXTargetFramework);net8.0</MinimalTargetFrameworksExeSigning>
    <MinimalTargetFrameworksExeSigning Condition=" '$(IsXPlat)' == 'true' ">net8.0</MinimalTargetFrameworksExeSigning>
```

---

My logic was to not depend on a preview framework just yet.

---

my opinion:  dev branch targets NuGet 6.9.  .NET 8 will go GA when 6.8 goes GA.  🤷